cmd/metal-api/internal/service/machine-service.go

```diff
@@ -1753,6 +1774,7 @@ func (r machineResource) deleteMachine(request *restful.Request, response *restf
 		}
 		delete(newIP.MachineConnections, m.ID)
 
+		// FIXME needs retry coverage, not so urgent, deleteMachine is called from metalctl, error is promoted and can be retried
```

Comment:
This is a pure operator endpoint, always manual and only on dead machines... no need for retry from my point of view.